js/songs/index.controller.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+(function(){
+  angular
+    .module("songs")
+    .controller("SongsIndexController", [
+//inject factory dependency
+      "SongFactory"
+      "$sce",
+      ControllerFunction
+    ])
+
+    function ControllerFunction(SongFactory, $sce) {
+//query for songs in the Controller Function
```

Comment:
You need `$sce` in order to play the songs, but since you have not defined `$sce.trustAsResourceUrl`, I would take either add that in to your code or take out the dependency